### ISSUE TITLE
ARIA hidden element must not be focusable or contain focusable elements

### ISSUE THREAD

**Issue by UnknownCoder404** [UnknownCoder404/Cubing-competition-nextjs#48]
Ensures aria-hidden elements are not focusable nor contain focusable elements. [Learn More.](https://dequeuniversity.com/rules/axe/4.9/aria-hidden-focus?application=Vercel%20Toolbar)

This happened on [dashboard](https://cubing-competition-nextjs.vercel.app/Dashboard).
